onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/IgnoredAlert.java

```diff
@@ -0,0 +1,45 @@
+package org.onebusaway.gtfs.model;
+
+import org.onebusaway.csv_entities.schema.annotations.CsvField;
+import org.onebusaway.csv_entities.schema.annotations.CsvFields;
+
+@CsvFields(filename = "ignored_alerts.txt", required = false)
+public final class IgnoredAlert extends IdentityBean<String> {
+
+  private static final long serialVersionUID = 2L;
+
+  @CsvField(name="alert_id")
+  private String id;
+
+  @CsvField(optional = true)
+  private String description;
+
+  public IgnoredAlert() {
+
+  }
+
+  public IgnoredAlert(IgnoredAlert a) {
+    this.id = a.id;
+    this.description = a.description;
```

Comment:
Do we care about parsing the description? Does it get used?

Reply:
No we're not using it at the moment but I just wanted our changes to be consistent with existing codebase (where they parse all the fields) so they look less like a hack.